proj_utils/Parser.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+from models.Payload import Payload
+import os
+import json
+
+
+class Parser:
+    def __init__(self, payload: Payload):
+        self.payload = payload
+        self.jsonObject = {}
+
+    def create_text_file(self):
+        save_path = 'C:/Users/razm1/PycharmProjects/routeExercise/students'
+        fullname = f"{self.jsonObject['studentDetails']['fullName']}"
+        complete_path = os.path.join(save_path, fullname.replace(' ', '_') + ".txt")
```

Comment:
what if the path to the students directory had a space in it too? this would not work.